go.mod

```diff
@@ -3,25 +3,24 @@ module github.com/evilsocket/shellz
 go 1.15
```

Comment:
mmm shouldn't this be 1.16?

Reply:
yes, this should be go1.16 as well. let me update it.